assets/assets.go

```diff
@@ -16,12 +17,12 @@ import (
 
 // initDocPaths lists the paths for the docs we want to seed during --init
 var initDocPaths = []string{
-	"init-doc/about",
-	"init-doc/readme",
-	"init-doc/help",
-	"init-doc/contact",
-	"init-doc/security-notes",
-	"init-doc/quick-start",
+	path.Join("init-doc", "about"),
+	path.Join("init-doc", "readme"),
+	path.Join("init-doc", "help"),
+	path.Join("init-doc", "contact"),
+	path.Join("init-doc", "security-notes"),
+	path.Join("init-doc", "quick-start"),
```

Comment:
thats odd... for some reason this made things work correctly on my windows vm. I'll switch it to `filepath` because thats the more correct way.